src/app/main.py

```diff
@@ -21,6 +21,18 @@ async def root():
     return "Hello World"
 
 
+@app.get("/expenses/", response_model=dict)
```

Comment:
this is the proper approach when type hinting, use the `Dict` instead of `dict`
```suggestion
@app.get("/expenses/", response_model=Dict[str, schemas.Expense])
```